src/erf.jl

```diff
@@ -20,3 +19,75 @@ function erfc(a::Interval{T}) where T
     isempty(a) && return a
     @round( erfc(a.hi), erfc(a.lo) )
 end
+
+function _erfinv(a::Float64)
+    domain = -1..1
+    a ∉ domain && return DomainError("$a is not in [-1, 1]")
+    rts = roots(x -> erf(x) - a, -Inf..Inf,
+        Newton(x->2/sqrt(pi_interval(Float64)) * exp(-x^2)))
+    @assert length(rts) == 1 && rts[1].status == :unique
+
+    rts[1].interval
+end
+
+function erfinv(a::BigFloat)
+    domain = Interval{BigFloat}(-1, 1)
+    a ∉ domain && return DomainError("$a is not in [-1, 1]")
+    rts = roots(x -> erf(x) - a, Interval{BigFloat}(-Inf, Inf),
+        Newton(x->2/sqrt(pi_interval(BigFloat)) * exp(-x^2)))
+    @assert length(rts) == 1 && rts[1].status == :unique
+
+    rts[1].interval
+end
+
+function erfinv(a::Interval{BigFloat})
+    domain = Interval{BigFloat}(-1, 1)
+    a = a ∩ domain
+
+    isempty(a) && return a
+    hull(erfinv(a.lo), erfinv(a.hi))
+end
+
+function erfinv(a::Interval{Float64})
+    domain = Interval{Float64}(-1, 1)
+    a = a ∩ domain
+
+    isempty(a) && return a
+    hull(_erfinv(a.lo), _erfinv(a.hi))
+end
+
+function _erfcinv(a::Float64)
+    domain = 0..2
+    a ∉ domain && return DomainError("$a is not in [-1, 1]")
+    rts = roots(x -> erfc(x) - a, -Inf..Inf,
+        Newton(x->-2/sqrt(pi_interval(Float64)) * exp(-x^2)))
+    @assert length(rts) == 1 && rts[1].status == :unique
+
+    rts[1].interval
+end
+
+function erfcinv(a::BigFloat)
```

Comment:
Similar comment for `erfcinv` -> `_erfcinv`.

Reply:
Actually, we should probably define `erfinv` and `erfcinv` of a `BigFloat` to return a `BigFloat` for consistency.

Reply:
I am not sure what to return though. Would it be correct to return the midpoint of the interval?

Reply:
Yes I think that's fine.

Reply:
If `erfinv(::BigFloat)` returns a number, do we need a `_erfinv(::Float64)`?

Reply:
Yes, sure.